src/encoded/static/components/item-pages/components/AttributionTabView.js

```diff
@@ -163,20 +163,26 @@ class LabsSection extends React.PureComponent {
     }
 
     render(){
-        var { context, className } = this.props,
-            isMounted           = this.state.mounted,
-            primaryLab          = (typeof context.lab !== 'string' && context.lab) || null,
-            contributingLabs    = ((Array.isArray(context.contributing_labs) && context.contributing_labs.length > 0) && context.contributing_labs) || null;
+        const {
+            context : {
+                lab: primaryLab = null,
+                contributing_labs: contributingLabs = []
+            },
+            className = null
+        } = this.props;
+        const contribLabLen = contributingLabs.length;
+        const { mounted } = this.state;
+
+        if (!primaryLab && contribLabLen === 0) return null;
 
-        if (!primaryLab && !contributingLabs) return null;
         return (
             <div className={className}>
-                { primaryLab ? FormattedInfoBlock.Lab(primaryLab, true, true, isMounted) : null }
-                { contributingLabs ?
+                { primaryLab ? FormattedInfoBlock.Lab(primaryLab, true, true, mounted) : null }
```

Comment:
Codacy found an issue: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5411404)